docs/extend/plugins.md

```diff
@@ -53,6 +53,10 @@ The following plugins exist:
   another volume plugin that provides multi-host volumes management for Docker
   using GlusterFS.
 
+* The [Horcrux Volume Plugin](https://github.com/muthu-r/horcrux)
+  An On Demand, Version controlled access to your Data. Horcrux is an open-source plugin,
```

Comment:
The "An On Demand" sentence is a bit weird, perhaps you can use the example I provided here; https://github.com/docker/docker/pull/20057#discussion_r52163446?